ci/script.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eux
 
-RUST_TOOLCHAIN="${RUST_TOOLCHAIN:-nightly-2022-11-14}"
+RUST_TOOLCHAIN="${RUST_TOOLCHAIN:-nightly-2023-02-07}"
```

Comment:
Do we know on which one parity is running polkadot and substrate?

Reply:
No, I just picked this since is the one for rust `v1.69`. I'll double-check what parity is using for their `v0.9.38` polkadot/substrate.

Reply:
I don't think parity is using any specific versions given:
https://github.com/paritytech/substrate/blob/polkadot-v0.9.38/.maintain/init.sh#L8

I also checked their CI scripts and it seems that they are using whatever is stable/nightly when they build, however, someone might know more. Maybe @wischli can help? ^^

Reply:
Parity doesn't use a nightly (I believe we only need it for fudge these days?), so I wouldn't be surprised if they just track stable.

Reply:
The releases in the Polkadot repository expose minimal compatible Rust versions For [v0.9.38](https://github.com/paritytech/polkadot/releases/tag/v0.9.38-rc12) that is

> * Rust Stable: rustc 1.66.1 (90743e729 2023-01-10)
> * Rust Nightly: rustc 1.67.0-nightly (a00f8ba7f 2022-11-15)

For the latest release [v0.9.42](https://github.com/paritytech/polkadot/releases/tag/v0.9.42) we are on

> * Rust Stable: rustc 1.68.2 (9eb3afe9e 2023-03-27)
> * Rust Nightly: rustc 1.67.0-nightly (a00f8ba7f 2022-11-15)

For their CI, they seem to follow latest stable as stated by @branan, here's an excerpt:

> [2023-05-30 22:10:38] $ cargo --version
> [2023-05-30 22:10:38] cargo 1.69.0 (6e9a83356 2023-04-12)

 However, in their Github actions, they are using [hongfuzz fuzzer with latest minimal nightly](https://github.com/paritytech/polkadot/blob/master/.github/workflows/honggfuzz.yml#L23).

Reply:
@branan